BLE/Protocol.cpp

```diff
@@ -1,4 +1,6 @@
 #include "Protocol.hpp"
+#include "ossia/network/base/node.hpp"
```

Comment:
The ossia style (which I4d recommend in general due to minimized ambiguousness is to use <> quotes always except for the main header that matches a .cpp file